### DISCUSSION TITLE
How do I create .img from ISO and deploy them

### DISCUSSION THREAD

**Discussion by DerekAppiahKumi** [ltsp/ltsp#325] [Q&A]
I have a 64 bit Server and three 32 bit ISO i want to make available to some very old thin Clients i have that only run 32 bit OSs, How do I turn those ISOs to img and make make them available on the server. I am aware there is a folder to pu them in and a config file but i don't know how to make the img accessible to my clients.

**Comment by alkisg** [comment]
You can just copy the iso to /srv/ltsp/images/x86_32.img (i.e. to rename its extension) and then run the following commands:

```shell
ltsp kernel x86_32
ltsp ipxe
```

**Reply by DerekAppiahKumi** [reply]
Thanks, it partially worked, so i was supposed to re name the .iso to .img
